R/Belgium.R

```diff
@@ -0,0 +1,215 @@
+#' Belgium Class for downloading, cleaning and processing notification data
+#'
+#' @description Country specific information for downloading, cleaning
+#'  and processing COVID-19 region level 1 and 2 data for Belgium.
+#'
+#' @details Inherits from `DataClass`
+#' @source https://opendata.arcgis.com/datasets/dd4580c810204019a7b8eb3e0b329dd6_0.csv  # nolint
+#' @export
+#' @examples
+#' \dontrun{
+#' region <- Belgium$new(verbose = TRUE, steps = TRUE, level = "2")
+#' region$download()
+#' region$clean()
+#' region$process()
+#' region$return()
+#' }
+Belgium <- R6::R6Class("Belgium",
+  inherit = DataClass,
+  public = list(
+
+    # Core Attributes
+    #' @field level_1_region the level 1 region name.
+    level_1_region = "region",
+    #' @field level_2_region the level 2 region name.
+    level_2_region = "province",
+    # nolint start
+    #' @field data_url link to raw cases data
+    data_url = "https://epistat.sciensano.be/Data/COVID19BE_CASES_AGESEX.csv",
+    #' @field hosp_url link to raw hospitalisation data
+    hosp_url = "https://epistat.sciensano.be/Data/COVID19BE_HOSP.csv",
+    #' @field deaths_url link to raw deaths data (mortality)
+    deaths_url = "https://epistat.sciensano.be/Data/COVID19BE_MORT.csv",
+    # nolint end
```

Comment:
These need refactoring inline with #221

Reply:
This has now been done, and the old custom download code removed.

I may have a got at extending the model to having separate data_urls for levels 1 and 2, but with the Belgium code we just download something we don't use at level 2.